### ISSUE TITLE
o-date: Incorrect use of aria - ID: DAC_Non_Text_Contrast_05

### ISSUE THREAD

**Issue by JakeChampion** [Financial-Times/origami#511]
# Issue
This is from page 52 of the [audit report](https://drive.google.com/file/d/17wq6b0CIX41RA4muBFDzUDxE0-qzHBIH/view)

Issue can be found on: <https://www.ft.com/__origami/service/build/v3/demo?component=o-date%405.0.1&demo=declarative&system_code=origami-repo-data&brand=master>

Two of the date/ time formats do not read accurately for screen reader users. An aria label has been used inside the element, but it has not been placed inside the <time> tag and is ineffective.

# Proposed Solution:
Ensure the date/ time formats are correctly structured and can be read correctly by screen reader users. Currently, the abbreviated time is read as 9h which may be confusing as the aria label is not supported on the time element.
A visually-hidden span or `<abbr>` may be readable by screen reader users, however we were unable to test this due to the clock updating the page.

**Comment by JakeChampion** [comment]
This looks to have been fixed in https://github.com/Financial-Times/origami/commit/43cbe14b164d48dc7f0b9e43fdade80b4d472549
